spring-integration-amqp/src/main/java/org/springframework/integration/amqp/dsl/RabbitStreamInboundChannelAdapterSpec.java

```diff
@@ -44,6 +44,42 @@ protected RabbitStreamInboundChannelAdapterSpec(Environment environment, @Nullab
 		super(new RabbitStreamMessageListenerContainerSpec(environment, codec));
 	}
 
+	/**
+	 * Configure a name for Rabbit stream to consume from.
+	 * @param streamName the name of Rabbit stream.
+	 * @return the spec
+	 * @since 6.1
+	 */
+	public RabbitStreamInboundChannelAdapterSpec streamName(String streamName) {
+		this.listenerContainerSpec.queueName(streamName);
+		return this;
+	}
+
+	/**
+	 * Configure a name for Rabbit super stream to consume from.
+	 * @param superStream the name of Rabbit super stream.
+	 * @param consumerName the logical name to enable offset tracking.
+	 * @return the spec
+	 * @since 6.1
+	 */
+	public RabbitStreamInboundChannelAdapterSpec superName(String superStream, String consumerName) {
+		return superName(superStream, consumerName, 1);
+	}
+
+	/**
+	 * Configure a name for Rabbit super stream to consume from.
+	 * @param superStream the name of Rabbit super stream.
+	 * @param consumerName the logical name to enable offset tracking.
+	 * @param consumers the number of consumers.
+	 * @return the spec
+	 * @since 6.1
+	 */
+	public RabbitStreamInboundChannelAdapterSpec superName(String superStream, String consumerName, int consumers) {
```

Comment:
The docs say this is `superStream` (which I think is better).

```suggestion
	/**
	 * Configure a name for Rabbit super stream to consume from.
	 * @param superStream the name of Rabbit super stream.
	 * @param consumerName the logical name to enable offset tracking.
	 * @return the spec
	 * @since 6.1
	 */
	public RabbitStreamInboundChannelAdapterSpec superStream(String superStream, String consumerName) {
		return superStream(superStream, consumerName, 1);
	}

	/**
	 * Configure a name for Rabbit super stream to consume from.
	 * @param superStream the name of Rabbit super stream.
	 * @param consumerName the logical name to enable offset tracking.
	 * @param consumers the number of consumers.
	 * @return the spec
	 * @since 6.1
	 */
	public RabbitStreamInboundChannelAdapterSpec superStream(String superStream, String consumerName, int consumers) {
```